.github/release-drafter.yml

```diff
@@ -26,8 +26,18 @@ categories:
   - title: '🧪 Testing'
     label: 'test'
 
-change-template: '- $BODY by @$AUTHOR'  # This will include the full PR description
-no-changes-template: "No notable changes in this release."
+change-template: '- $TITLE (#$NUMBER) @$AUTHOR'
 
-# Optional customization to control how the version is incremented
-version-template: $MAJOR.$MINOR.$PATCH
+template: |
+  ## What's Changed
+
+  ### Detailed Changes
+  {% for change in changes %}
+  - **Title**: {{ change.title }}
+  - **PR**: [#{{ change.number }}]({{ change.html_url }})
+  - **Author**: @{{ change.author.login }}
+  - **Motivation**:
+    {{ change.body | paragraph_start_with("# Motivation") | default("No specific motivation provided.") }}
```

Comment:
Will this get the whole markdown section, or just the first paragraph in the section?

Reply:
I couldn't find the documentation for `paragraph_start_with` to confirm. Do you know where that is?

Reply:
There is no real documentation on that. Basically the release-drafter.yml uses Handlebars as template-engine. But paragraph_start_with is not part of it. So I want to try and see what happens. Otherwise I feel like I can't really get just the Motivation or just the Changes.